src/pages/issues/index.tsx

```diff
@@ -82,6 +112,12 @@ const Issues: FC = () => {
         }
     };
 
+    useEffect(() => {
+        if (dev === 'true' && splittedQueryParam) {
+            fetchIssues(splittedQueryParam);
```

Comment:
`splittedQueryParam` name does not suggest it is the search term. Naming can be improved.

Reply:
done